### PR TITLE
Feature/migration files

### PR DESCRIPTION
**Pull request by Luiscarlosvd**
### In this milestone

- Create a new app. Give it a meaningful name. ✔️ 
- Initialize your project with Git. ✔️ 
- Make sure that your project has Postgres database set up. ✔️ 
- Create migration files. ✔️ 
- Table and column names should match the ERD diagram. ✔️ 
- Foreign keys should be included. ✔️ 
- Pay attention that in posts table the column holding the foreign key to the users table should be named author_id. ✔️ 
- All columns that are foreign keys should have a corresponding index. ✔️ 

**_Authors: 
@Luiscarlosvd_**

### CODE REVIEW

File: db/migrate/20230815193556_add_ref_users_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddRefUsersToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :author, null: false, foreign_key: true
+    add_index :posts, :author
```
Comment (julie-ify) [inline db/migrate/20230815193556_add_ref_users_to_posts.rb]: - [x] The `foreign_key` attribute should refer to the users, even if the column id would be `author_id`

  ```
  ...foreign_key: { to_table: :users }
  ```

  image

- [x] Kindly remove this `add_index` line as `add_reference` above already adds the index.

File: db/migrate/20230815195144_add_ref_users_to_comments.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class AddRefUsersToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :author, null: false, foreign_key: true
+    add_index :comments, :author
```
Comment (julie-ify) [inline db/migrate/20230815195144_add_ref_users_to_comments.rb]: - [x] The `foreign_key` attribute should refer to the users, even if the column id would be `author_id`

  ```
  ...foreign_key: { to_table: :users }
  ```

- [x] Kindly remove this `add_index` line as `add_reference` above already adds the index.

File: db/migrate/20230815193949_create_comments.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :post, null: false, foreign_key: true
+
+      t.timestamps
+    end
+    add_index :comments, :post
```
Comment (julie-ify) [inline db/migrate/20230815193949_create_comments.rb]: - [x] Kindly remove the `add_index` line. The references line you have above takes care of this.

File: db/migrate/20230815200321_add_ref_users_to_likes.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddRefUsersToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :author, null: false, foreign_key: true
```
Comment (julie-ify) [inline db/migrate/20230815200321_add_ref_users_to_likes.rb]: - [x] The `foreign_key` attribute should refer to the users, even if the column id would be `author_id`

  ```
  ...foreign_key: { to_table: :users }
  ```

- [x] After fixing these issues you still need to migrate your database schema and set up the `db/schema.rb` file, which is absent currently. Kindly run the command below.

  ```
  rails db:migrate
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Luiscarlosvd,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aamirkhan2478 — APPROVED** [review]
Hi @Luiscarlosvd, 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**